src/main/java/thePackmaster/actions/siegepack/BallisticStrikeAction.java

```diff
@@ -0,0 +1,45 @@
+package thePackmaster.actions.siegepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.DarkSmokePuffEffect;
+import com.megacrit.cardcrawl.vfx.combat.ExplosionSmallEffect;
+import com.megacrit.cardcrawl.vfx.combat.WeightyImpactEffect;
+import thePackmaster.util.Wiz;
+
+//REF: Alchyr code, StaticAction (distortionpack), others
+public class BallisticStrikeAction extends AbstractGameAction {
+    private final AbstractCard card;
+    private static final float VFX_X_OFFSET = 120F;   //it's seemingly pixels.
```

Comment:
You need to multiply pixel adjustments like these by Settings.scale, otherwise it'll look bad on other resolutions
```suggestion
    private static final float VFX_X_OFFSET = 120F * Settings.scale;
```